README.md

```diff
@@ -246,6 +246,15 @@ allowedClasses: {
 }
 ```
 
+Furthermore, regex expressions are supported via strings that start with `^` and end with `$`:
```

Comment:
Other options already allow actual regular expression objects. So for consistency you should do that rather than looking for certain strings and converting them to regular expression objects.

Reply:
I definitely did not took the cleanest way to add this functionality but rather the fastest one and I _think_ the most performant (no new declarations or new conversions rather than the existing ones) -I mostly wanted to 'test the waters' and see if this was needed/welcomed.
Will take a look at allowedStyles where I've seen regExp used in declaration and will update 👍

Reply:
Please take look to see if we are aligned with these changes.